plugins/woocommerce/src/Admin/API/OnboardingPlugins.php

```diff
@@ -246,17 +246,20 @@ public function get_jetpack_authorization_url( WP_REST_Request $request ) {
 			$calypso_env = 'horizon';
 		}
 
-		return [
-			'success' => ! $errors->has_errors(),
-			'errors'  => $errors->get_error_messages(),
-			'url'     => add_query_arg(
+		$color_scheme = get_user_option( 'admin_color', get_current_user_id() );
```

Comment:
Should this have a default value? If say the `admin_color` meta is deleted (I used `wp user meta delete 1 admin_color`), I believe this will be null.

Reply:
@ilyasfoo Good point. I have a wp-calypso PR https://github.com/Automattic/wp-calypso/pull/96557 that ignores any invalid values and falls back to the default.

I think it's still a good idea to set the default.

Updated it in [Set color scheme to default if it is missing](https://github.com/woocommerce/woocommerce/pull/52925/commits/3faf67cd628f206327842eb680c03d4d2e410ba4)